danielssonn/ethTO
# feat: ui updates

## Purpose

1. Removed `NftCard` and `NftModal`
2. Refactored the `NftModal` to `SwingModal`
3. Updated the `Arrivals` and `Departures` pages UI to a table
4. Removed the Swing step from `Checkout`
5. Added a basic GitHub action (deploy step needs to be added)
6. Updated anchors to `<Link>`

## Testing Instructions

1. Confirm the Swing modal and the Arrivals, Checkout and Departures pages look correct (optional):

- Pull down the branch
- Move into the `client` directory and run `npm run dev`
- Navigate to `http://localhost:3000/arrivals`
- Click on Swap tokens
- Navigate to `http://localhost:3000/checkout/0x123b30e25973fecd8354dd5f41cc45a3065ef88c/1` (this is just a random contract)
- Navigate to `http://localhost:3000/departures`

2. Review code

## Checklist
